pkg/tenancy/tenancy.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package tenancy
+
+import (
+	"net/http"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	// DefaultTenantHeader is the default header used to designate the tenant making a write request.
+	DefaultTenantHeader = "THANOS-TENANT"
+	// DefaultTenant is the default value used for when no tenant is passed via the tenant header.
+	DefaultTenant = "default-tenant"
+	// DefaultTenantLabel is the default label-name used for when no tenant is passed via the tenant header.
```

Comment:
This comment seems to be a bit wrong (was wrong before too). Let's update.
```suggestion
	// DefaultTenantLabel is the default label-name with which the tenant is announced in stored metrics.
```